peekaboo/daemon.py

```diff
@@ -211,13 +210,17 @@ def run():
                                    stat.S_IWRITE | stat.S_IRGRP |
                                    stat.S_IWGRP | stat.S_IWOTH)
 
-        # Run Cuckoo sandbox, parse log output, and report back of Peekaboo.
-        # If this dies Peekaboo dies, since this is the main thread.
-        srv = CuckooServer()
-        reactor.spawnProcess(srv, config.interpreter, [config.interpreter, '-u',
-                                                       config.cuckoo_exec])
+        # If this dies Peekaboo dies, since this is the main thread. (legacy)
+        if config.cuckoo_mode == "embed":
+            cuckoo = CuckooEmbed(config.interpreter, config.cuckoo_exec)
+        elif config.cuckoo_mode == "dummy":
+            cuckoo = CuckooDummy()
```

Comment:
@Jack28 you forgot to remove ``CuckooDummy`` from ``daemon.py``.

Reply:
I've noticed that. There is a commit on my fork/branch that fixes this. How do you want me to include this?
Revert and merge again, commit to master ... ?